jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1718,6 +1718,30 @@ def _eigh(operand: TfVal, lower: bool):
 
 tf_impl[lax_linalg.eigh_p] = _eigh
 
+def _triangular_solve(a: TfVal, b: TfVal, *, left_side: bool, lower: bool,
+                      transpose_a: bool, conjugate_a: bool,
+                      unit_diagonal: bool):
+  if unit_diagonal:
+    a = (a - tf.linalg.diag(tf.linalg.diag_part(a)) +
```

Comment:
This would not be correct if there are NaNs on the diagonal.

Reply:
Thanks Peter! `d840c34` should fix this!